packages/nc-gui-v2/components/dlg/QuickImport.vue

```diff
@@ -18,7 +18,7 @@ interface Props {
   importType: 'csv' | 'json' | 'excel'
 }
 
-const { modelValue, importType } = defineProps<Props>()
+const props = defineProps<Props>()
```

Comment:
If you need `props` to stay an object for `useVModel` but would still like to destructure parts of it, you can use this

```ts
const { importType, ...rest } = defineProps<Props>()

const dialogShow = useVModel(rest, 'modelValue', emit)
```